target/consensource-ui
# Create default landing page

Signed-off-by: Patrick-Erichsen <Patrick.Erichsen@target.com>

## Proposed change/fix

Creates a landing page with links to each view/application. This prevents users from needing to know the correct `.html` file that will serve each app.

## Types of changes

What types of changes does this pull request introduce to ConsenSource? _Put an `x` in the boxes that apply_


## How to run/test

- `npm run watch` and visit `localhost:8080` - confirm that each link opens a tab with the correct application.
